lua/bookmarks/config.lua

```diff
@@ -13,7 +13,11 @@ M.schema = {
    },
    save_file = {
       type = "string",
-      default = vim.fn.expand "$HOME/.bookmarks",
+      default = vim.fn.stdpath("state") .. "/bookmarks",
```

Comment:
This avoids polluting the users home directory with `.bookmarks` that you may not even know where it is coming from.

Additionally, this directory change avoids errors from existing global bookmarks, as the path is now a directory